modules/stochastic_tools/src/utils/AdaptiveMonteCarloUtils.C

```diff
@@ -0,0 +1,42 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "MooseUtils.h"
+#include "AdaptiveMonteCarloUtils.h"
+
+/* AdaptiveMonteCarloUtils contains functions that are used across the Adaptive Monte
+Carlo set of algorithms.*/
+
+// Compute standard deviation of a data vector by ignoring some values in the vector at the beginning
+Real
+AdaptiveMonteCarloUtils::computeSTD(const std::vector<Real> & data, const unsigned int start_index)
+{
+  Real sum1 = 0.0, sq_diff1 = 0.0;
+  for (unsigned int i = start_index; i < data.size(); ++i)
+  {
+    sum1 += (data[i]);
+  }
+  for (unsigned int i = start_index; i < data.size(); ++i)
+  {
+    sq_diff1 += std::pow(((data[i])-sum1/(data.size()-start_index)), start_index);
```

Comment:
```suggestion
    sq_diff1 += std::pow((data[i]-sum1/(data.size()-start_index)), start_index);
```
power of start_index? This doesnt seem right. Can you please check

Reply:
Yes, this should be 2. Thanks for pointing out.